identity/app/conf/IdentityConfiguration.scala

```diff
@@ -20,4 +21,17 @@ class IdentityConfiguration extends GuardianConfiguration with SafeLogging {
     }
   }
 
+  // Userhelp - 'Guardian Admin' ET Account integrated with subscriptions.dev@guardian.co.uk AppCentre
+  object exactTargetUserhelp {
```

Comment:
Bringing this sort of behaviour into Frontend account isn't in line with the expectations of privacy and security for that AWS account. Generally applications in Frontend are considered simple transformations from JSON to HTML, whether that's from the Identity API or more commonly, the content API.

I'm sure this isn't the only example of other behaviour, but in this case you're bringing into the Frontend account the ability to *directly manage* the personal data of our users. I'd want to have a chat with the wider Frontend team about how they feel taking on the burden of maintaining the security of such a service.